pkg/reconciler/openshift/tektonresult/extension.go

```diff
@@ -37,17 +37,20 @@ import (
 
 const (
 	// manifests console plugin yaml directory location
-	routeRBACYamlDirectory  = "static/tekton-results/route-rbac"
-	logsRBACYamlDirectory   = "static/tekton-results/logs-rbac"
-	deploymentAPI           = "tekton-results-api"
-	serviceAPI              = "tekton-results-api-service"
-	routeAPI                = "tekton-results-api"
-	secretAPITLS            = "tekton-results-tls"
-	apiContainerName        = "api"
-	boundSAVolume           = "bound-sa-token"
-	boundSAPath             = "/var/run/secrets/openshift/serviceaccount"
-	lokiStackTLSCAEnvVar    = "LOGGING_PLUGIN_CA_CERT"
-	tektonResultWatcherName = "tekton-results-watcher"
+	routeRBACYamlDirectory     = "static/tekton-results/route-rbac"
+	logsRBACYamlDirectory      = "static/tekton-results/logs-rbac"
+	deploymentAPI              = "tekton-results-api"
+	serviceAPI                 = "tekton-results-api-service"
+	routeAPI                   = "tekton-results-api"
+	secretAPITLS               = "tekton-results-tls"
+	apiContainerName           = "api"
+	boundSAVolume              = "bound-sa-token"
+	boundSAPath                = "/var/run/secrets/openshift/serviceaccount"
+	lokiStackTLSCAEnvVar       = "LOGGING_PLUGIN_CA_CERT"
+	tektonResultWatcherName    = "tekton-results-watcher"
+	postgresStatefulSetName    = "tekton-results-postgres"
+	postgresContainerName      = "postgres"
+	postgresUpgradeScriptsName = "postgres-upgrade-scripts"
```

Comment:
I see these constants are in 2 places can we keep it in common as its common to Openshift. + Kubernetes

Reply:
They are only required in OCP. I removed them from the k8s file.